SharpGenTools.Sdk/SharpGenTools.Sdk.csproj

```diff
@@ -35,7 +35,6 @@
     <PackageReference Include="Microsoft.Bcl.HashCode" />
     <PackageReference Include="System.Text.Json" />
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" />
-    <PackageReference Include="Polly" />
```

Comment:
[Polly](https://github.com/App-vNext/Polly) is a great project, but feels overengineered and yet still too complex to customize retry/fallback behavior. I dropped the dependency. `Polly.Contrib.WaitAndRetry` (despite the name) doesn't depend on `Polly` and provides good exponential backoff formula.